recipes/clickhouse-cityhash/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: clickhouse-cityhash
+  version: "1.0.2"
+
+source:
+  fn: clickhouse-cityhash-1.0.2.tar.gz
+  url: https://pypi.python.org/packages/fb/e3/282f812aeace32817a6b2b894ccdfbe31146364294668cc85e1ad87b6f3f/clickhouse-cityhash-1.0.2.tar.gz
+  md5: 13982de613981509c68377d328238304
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
It looks like this builds C-extensions so please add `toolchain` as a `build` requirement.

Reply:
Added.